test/acceptance/features/bindAppToServiceUsingConfigMap.feature

```diff
@@ -19,7 +20,7 @@ Feature: Bind values from a config map referred in backing service resource
             metadata:
                 name: backends.stable.example.com
                 annotations:
-                    service.binding/username: path={.status.data.dbConfiguration},objectType=ConfigMap,valueKey=username
+                    service.binding/username: path={.status.data.dbConfiguration},objectType=ConfigMap,sourceKey=username
```

Comment:
we should expect `BACKEND_USERNAME` env var in the application context, not `BACKEND_CERTIFICATE`, right?

Reply:
Sorry it's `BACKEND_CERTIFICATE` I need to rename the username to certificate.